app/facades/cocktail_facade.rb

```diff
@@ -1,21 +1,53 @@
 class CocktailFacade
-
-  def self.three_random
-    cocktails = CocktailService.get_three_random
+  def self.by_name(query)
+    cocktails = CocktailService.get_search_results(query)[:drinks]
     cocktails.map do |cocktail|
       ingredients = []
       15.times do |i|
-        if cocktail[:drinks][0]["strIngredient#{i+1}".to_sym]
-          name = cocktail[:drinks][0]["strIngredient#{i+1}".to_sym]
-          measurement = cocktail[:drinks][0]["strMeasure#{i+1}".to_sym]
+        if cocktail["strIngredient#{i+1}".to_sym]
+          name = cocktail["strIngredient#{i+1}".to_sym]
+          measurement = cocktail["strMeasure#{i+1}".to_sym]
           ingredients << { name: name, measurement: measurement }
         end
       end
       ingredients.map! do |ingredient|
         Ingredient.new(name: ingredient[:name], quantity: ingredient[:measurement])
       end
-      Drink.new(id: cocktail[:drinks][0][:idDrink], name: cocktail[:drinks][0][:strDrink], img_url: cocktail[:drinks][0][:strDrinkThumb], steps: cocktail[:drinks][0][:strInstructions], ingredients: ingredients)
```

Comment:
We need to check with FE on what size image they want/need so we are returning the correct image on all the API calls. 
- Everything else looks good, not merging yet to use this PR as a FE discussion reminder.

Reply:
Keeping this image it works for FE and is only consistently available one